client/src/components/DataFiles/DataFilesModals/DataFilesMkdirModal.jsx

```diff
@@ -58,22 +64,29 @@ const DataFilesMkdirModal = () => {
           validationSchema={validationSchema}
           onSubmit={mkdirCallback}
         >
-          <Form>
-            <ModalHeader toggle={toggle} charCode="&#xe912;">
-              Creating folder in {systemDisplayName}/{params.path}
-            </ModalHeader>
-            <ModalBody>
-              <FormField
-                name="dirname"
-                label="Enter a name for the new folder:"
-              />
-            </ModalBody>
-            <ModalFooter>
-              <Button type="primary" size="long" attr="submit">
-                Create Folder
-              </Button>
-            </ModalFooter>
-          </Form>
+          {({ isSubmitting }) => (
+            <Form>
+              <ModalHeader toggle={toggle} charCode="&#xe912;">
+                Creating folder in {systemDisplayName}/{params.path}
+              </ModalHeader>
+              <ModalBody>
+                <FormField
+                  name="dirname"
+                  label="Enter a name for the new folder:"
+                />
+              </ModalBody>
+              <ModalFooter>
+                <Button
+                  type="primary"
+                  size="long"
+                  attr="submit"
+                  disabled={isSubmitting}
+                >
+                  {!isSubmitting ? 'Create Folder' : <Spinner size="sm" />}
+                </Button>
```

Comment:
The `<Button>` component now has a handy `isLoading` prop to handle the inline `<Spinner>`: https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/Button/Button.jsx#L48

Can we use that prop here?

Reply:
updated code to use isLoading to handle the spinner